pkg/resolver/client/ens/ens.go

```diff
@@ -5,19 +5,23 @@
 package ens
 
 import (
+	"bytes"
 	"errors"
 	"fmt"
 	"strings"
 
-	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/ethclient"
 	goens "github.com/wealdtech/go-ens/v3"
 
 	"github.com/ethersphere/bee/pkg/resolver/client"
 	"github.com/ethersphere/bee/pkg/swarm"
 )
 
-const swarmContentHashPrefix = "/swarm/"
+const (
+	defaultENSContractAddress = "00000000000C2E074eC69A0dFb2997BA6C7d2e1e"
```

Comment:
NOTE: this is the default ENS registry address. If the ENS suite is deployed from the current builds to any chain, it will always be found at this address.